templates/cli/lib/commands/init.js.twig

```diff
@@ -40,12 +40,35 @@ const initProject = async () => {
         log('You must login first')
         await loginCommand();
     }
+    let answers = {};
 
-    const answers = await inquirer.prompt(questionsInitProject)
-    if (answers.override === false) {
-        process.exit(1)
+    if (organizationId && projectId && projectName) {
```

Comment:
I think we should add ability to set organization ID, and nothing else. That way you are still asked questions, but not about organization. I think this would make sense for each param.